chef/cookbooks/bind9/templates/default/named.conf.erb

```diff
@@ -39,8 +39,13 @@ options {
         };
 <% end -%>
         auth-nxdomain no;    # conform to RFC1035
+<% if @enable_designate and !@publicaddress.nil? -%>
+        listen-on { <%= @ipaddress %>; <%= @publicaddress %>; };
+        listen-on-v6 { <%= @ip6address %>; <%= @public6address%>; };
+<% else -%>
```

Comment:
I wonder if we want to switch to an array and then you can just ipaddrs.join('; ') and you can stop passing enable_designate into the template, and the smarts remain in the recipe.

Reply:
Great idea!! That simplifies it greatly. And then the old dns_master + enable_designate logic remains untouched. You sir are a genius!